tests/queries/test_qs_combinators.py

```diff
@@ -258,3 +258,13 @@ def test_order_by_same_type(self):
         numbers = list(range(10))
         self.assertNumbersEqual(union.order_by('num'), numbers)
         self.assertNumbersEqual(union.order_by('other_num'), reversed(numbers))
+
+    def test_unsupported_operations(self):
+        qs = Number.objects.all()
+        msg = 'Calling %s() is not supported after %s().'
+        for combinator in ('union', 'intersection', 'difference'):
+            for method in ('filter', 'exclude', 'defer', 'only', 'extra',
+                           'prefetch_related', 'select_related'):
```

Comment:
We should add `annotate()` to the list, also please use hanging indentation.

Reply:
`annotate()` added.